### PR TITLE
4.0.0

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: vulcanexus.repos
Comment (Mario-DL) [inline vulcanexus.repos:85]:
```suggestion
    version: v3.1.0
```

**Review by Mario-DL — CHANGES_REQUESTED** [review]
I guess that we will also wait for the releases from `Fast DDS Monitor`  and `Fast DDS Statistics Backend` to be published before merging

**Review by Mario-DL — APPROVED** [review]
LGTM

**Comment by Mario-DL** [comment]
I dont know whether we should add the release notes under `docs/notes/`

File: docs/rst/installation/linux_binary_installation.rst
Comment (Mario-DL) [inline docs/rst/installation/linux_binary_installation.rst:6]:
```suggestion
Debian packages for Vulcanexus jazzy Jolo are currently available for Ubuntu Jammy.
```

File: docs/rst/platforms/platforms.rst
Comment (Mario-DL) [inline docs/rst/platforms/platforms.rst:23]:
```suggestion
    * - :ref:`Jazzy Jolo <notes_jazzy_latest>` v4
```

File: docs/rst/tutorials/core/deployment/ds_wan_tcp/ds_wan_tcp.rst
Comment (Mario-DL) [inline docs/rst/tutorials/core/deployment/ds_wan_tcp/ds_wan_tcp.rst:131]:
```suggestion
The :ref:`Vulcanexus Jazzy <notes_jazzy_latest>` image can be downloaded by running:
```

File: docs/rst/notes/jazzy/notes.rst
Comment (Mario-DL) [inline docs/rst/notes/jazzy/notes.rst:1]:
```suggestion
.. _notes_jazzy_latest:
```

File: docs/conf.py
Comment (Mario-DL) [inline docs/conf.py:100]: Should this reference be updated ?

File: docs/conf.py
Comment (Mario-DL) [inline docs/conf.py:98]: Should this be `Jazzy Jolo` ?

**Review by Mario-DL — CHANGES_REQUESTED** [review]

File: docs/rst/installation/linux_binary_installation.rst
Comment (Mario-DL) [reply docs/rst/installation/linux_binary_installation.rst:6]: NIT: additionally in capital `Jazzy`

File: docs/rst/platforms/platforms.rst
Comment (rsanchez15) [reply docs/rst/platforms/platforms.rst:23]: Added a new line with jazzy

File: docs/conf.py
Comment (rsanchez15) [reply docs/conf.py:100]: Yes

File: docs/conf.py
Comment (rsanchez15) [reply docs/conf.py:98]: Yes

**Review by Mario-DL — APPROVED** [review]

**Pull request merged**
